src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,545 @@
+    alt="Demo web application for punch speed measurement"
+    class="w-screenshot"
+  >
+  <figcaption class="w-figcaption">Measurement of a punch speed</figcaption>
+</figure>
+
+## Debugging and sensor overriding with Chrome DevTools
+
+In some cases you don't need a physical device to play with the Generic Sensor API. Chrome DevTools
+has great support for
+[simulating device orientation](https://developers.google.com/web/tools/chrome-devtools/device-mode/orientation).
+
+<figure class="w-figure">
+  <img  src="orientation-model.png"
+    alt="Chrome DevTools used to override the custom orientation data of a virtual phone.">
+  <figcaption class="w-figcaption">Simulating device orientation with Chrome DevTools</figcaption>
+</figure>
+
+## Privacy and security {: #privacy-and-security }
+
+Sensor readings are sensitive data which can be subject to various attacks from malicious web pages.
+Implementations of Generic Sensor APIs enforce a few limitations to mitigate the possible security
+and privacy risks. These limitations must be taken into account by developers who intend to use the
+API, so let's briefly list them.
+
+### Only HTTPS
+
+Because Generic Sensor API is a powerful feature, the browser only allows it on secure contexts. In
+practice it means that to use Generic Sensor API you'll need to access your page through HTTPS.
+During development you can do so via [http://localhost](http://localhost) but for production you'll
+need to have HTTPS on your server. See the [Safe and secure](/secure/) collection for best practices
+and guidelines.
+
+### Permissions Policy integration
+
+The [Permissions Policy integration](https://w3c.github.io/webappsec-permissions-policy/) in Generic
+Sensor API controls access to sensors data for a frame.
+
+By default the `Sensor` objects can be created only within a main frame or same-origin subframes,
+thus preventing cross-origin iframes from unsanctioned reading of sensor data. This default behavior
+can be modified by explicitly enabling or disabling of the corresponding
+[policy-controlled features](https://w3c.github.io/webappsec-permissions-policy/#features).
+
+The snippet below illustrates granting accelerometer data access to a cross-origin iframe, meaning
+that now `Accelerometer` or `LinearAccelerationSensor` objects can be created there.
+
+```html
+<iframe src="https://third-party.com" allow="accelerometer" />
+```
+
+### Sensor readings delivery can be suspended
+
+Sensor readings are only accessible by a visible web page, i.e., when the user is actually
+interacting with it. Moreover, sensor data would not be provided to the parent frame if the user
+focus changes to a cross-origin subframe. This prevents the parent frame infering user input.
+
+## What's next? {: #whats-next }
+
+There is a set of already specified sensor classes to be implemented in the near future such as
+[Proximity sensor](https://w3c.github.io/proximity/); however, thanks to the great extensibility of
```

Comment:
Is this still being worked on?
I would emphasize AmbientLightSensor and GravitySensor instead.

Reply:
Gravity is almost there, so I list ALS and Proximity for now.